html/manifest/display_override.json

```
@@ -0,0 +1,55 @@
+{
+  "html": {
+    "manifest": {
+      "display_override": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/Manifest/display_override",
+          "spec_url": "https://wicg.github.io/manifest-incubations/#display_override-member",
+          "support": {
+            "chrome": {
+              "version_added": "89"
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "edge": {
+              "version_added": "89"
+            },
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": null
```

```suggestion
              "version_added": "75"
```

According to the [release notes](https://help.opera.com/en/opera-version-history/#:~:text=Version%20history-,Opera%2075,Chromium%2089,-JavaScript%20engine).